dspace-api/src/main/java/org/dspace/content/Collection.java

```diff
@@ -52,7 +55,7 @@
 @Table(name = "collection")
 @Cacheable
 @org.hibernate.annotations.Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE, include = "non-lazy")
-public class Collection extends DSpaceObject implements DSpaceObjectLegacySupport {
+public class Collection extends DSpaceObject implements DSpaceObjectLegacySupport, BrowsableDSpaceObject<UUID> {
```

Comment:
Since `DSpaceObject` already implements `BrowsableDSpaceObject<UUID>`, is this change still necessary?   I see the same is true for `Community` and `Item`, as they both implement this interface, even though it's already implemented by `DSpaceObject`.  Are these duplicate `implements` necessary?

Reply:
you are right, it is unnecessary. Removed